### PR TITLE
Bug Fix for decodeValveData / improvedNaming / CIRCUIT_VALUE_MAP

### PR DESCRIPTION
**Pull request by bshep**

### CODE REVIEW

File: messages/SLEquipmentConfigurationMessage.js [hunk 1]
```diff
@@ -198,22 +199,24 @@ exports.SLEquipmentConfigurationMessage = class SLEquipmentConfigurationMessage
...
   }
 
   decodeMiscData() {
     this.misc = {};
 
     if (this.isBitSet(this.miscDataArray[3], 0) === false) {
       this.misc.intelliChem = false;
+    } else {
+      this.misc.intelliChem = true;
     }
```
Comment (parnic) [inline messages/SLEquipmentConfigurationMessage.js:214]: This entire thing could be collapsed to
`this.misc.intelliChem = this.isBitSet(this.miscDataArray[3], 0);`

File: messages/SLEquipmentConfigurationMessage.js [hunk 2]
```diff
@@ -198,22 +199,24 @@ exports.SLEquipmentConfigurationMessage = class SLEquipmentConfigurationMessage
...
 
     if (this.miscDataArray[4] !== 0) {
-      this.misc.spaManual = true;
+      this.misc.spaManualHeat = true;
     } else {
-      this.misc.spaManual = false;
+      this.misc.spaManualHeat = false;
     }
```
Comment (parnic) [inline messages/SLEquipmentConfigurationMessage.js:220]: Likewise here,
`this.misc.spaManualHeat = this.miscDataArray[4] !== 0;`

File: messages/SLControllerConfigMessage.js [hunk 3]
```diff
@@ -4,6 +4,14 @@ const SLMessage = require('./SLMessage.js').SLMessage;
 
 const MSG_ID = 12532;
 
+const CIRCUIT_NAME_VALUE_MAP = [
+  ['Unused', 0],
+  ['Solar Active', 128],
+  ['Pool or Spa Heater Active', 129],
+  ['Pool Heater Active', 130],
+  ['Spa Heater Active', 131],
+];
```
Comment (bshep) [inline messages/SLControllerConfigMessage.js:13]: I think this should be in the form of:
```[{name:'Unused', deviceId: 128},...]```

That way we can 'concat' with the bodyarray and simplify the getCircuitById function.  Thoughts?

Comment (parnic) [reply messages/SLControllerConfigMessage.js:13]: Sounds good

**Pull request merged**